packages/web-runtime/src/App.vue

```diff
@@ -220,6 +220,10 @@ export default {
       return 'uk-visible@l'
     },
 
+    isSidebarItems() {
+      return this.sidebarNavItems.length > 0
```

Comment:
Please add
```js
if (this.sidebarNavItems.length === 0) {
    return false
}
```
to `isSidebarVisible` instead.

Reply:
Would like to move towards `early returns` in code style.

Reply:
Code is updated to reflect this 🤓